syntax = "proto3";

package gameSvr;


enum Direction {
    DIRECTION_NORTH         = 0;
    DIRECTION_SOUTH         = 1;
    DIRECTION_WEST          = 2;
    DIRECTION_EAST          = 3;
}


enum AttrType {
    ATTR_NONE               = 0;
    ATTR_HP                 = 1;
    ATTR_DAMAGE             = 2;
}
